src/routes/transactions/mappers/multisig-transactions/multisig-transaction-execution-details.mapper.ts

```diff
@@ -42,6 +42,9 @@ export class MultisigTransactionExecutionDetailsMapper {
               confirmation.submissionDate.getTime(),
             ),
         );
+    const proposer = transaction.proposer
+      ? new AddressInfo(transaction.proposer)
+      : null;
```

Comment:
Do we need to return the full Address Info if we have it? (implies calling the `AddressInfoHelper` instead).

Reply:
Adding that would result in an another request. As the client is (currently) only using this as a flag, I don't think it's needed. We can always add it later if necessary. What do you think?

Reply:
> Adding that would result in an another request. As the client is (currently) only using this as a flag, I don't think it's needed. We can always add it later if necessary. What do you think?

Sounds good 👍 We can it later if the requirements change.